scripting/franug_weaponpaints_private.sp

```diff
@@ -1546,25 +1553,28 @@ Nuevo(client)
 
 public PruneDatabase()
 {
-	if (db == INVALID_HANDLE)
+	if (g_iInactivedays != -1)
```

Comment:
This looks weird on the commit but its just wrapping the database pruning in an if statement to check if the g_iInactivedays is not set to -1